locator/util.cc

```diff
@@ -147,4 +147,12 @@ describe_ring(const replica::database& db, const gms::gossiper& gossiper, const
     co_return ranges;
 }
 
+bool dc_is_arbiter(sstring_view datacenter) {
+    static const sstring suffix = "__arbiter__";
+    if (suffix.length() > datacenter.length()) {
+        return false;
+    }
+    return std::equal(suffix.cbegin(), suffix.cend(), datacenter.cend() - suffix.length());
+}
```

Comment:
I believe we should move towards making datacenters (and racks) first class objects rather than an attribute that is attached to nodes.

e.g.

```cql
CREATE DATACENTER dc1;

CREATE RACK dc1_rack1 WITH DATACENTER = dc1;

...
```

Then we can attach any property we want to a datacenter.

Although, that conflicts with datacenter auto-detection using snitches. Maybe we can think of a way to combine them.

In any case, out of scope for the series.

Reply:
How does the user choose the name with ec2_snitch? It will pick up the datacenter name from ec2 metadata.

Reply:
They can specify `dc_suffix` in the property file.

Reply:
Very good, did not know that. Please make it part of the how-to documentation.